main/src/ca/uwaterloo/flix/language/phase/typer/Debug.scala

```diff
@@ -136,6 +136,8 @@ object Debug {
     */
   private def toSubDot(constr: TypeConstraint): String = constr match {
     case TypeConstraint.Equality(tpe1, tpe2, _) => s"""${dotId(constr)} [label = "$tpe1 ~ $tpe2"];"""
+    case TypeConstraint.EqJvmConstructor(mvar, clazz, _, _) => s"TODO JVMCONSTRUCTOR" // TODO INTEROP graphviz representation
```

Comment:
@mlutze Thoughts

Reply:
Thank you. You can check the update on this code section to see if it is a valid graphviz representation.